app/views/users/settings.html.erb

```diff
@@ -1,6 +1,12 @@
 <% content_for :title, t('students.account_settings') %>
 <% if @current_user.student? %>
-  <%= form_with url: 'update_mailer_settings', method: 'patch', model: @current_user, local: true do |f| %>
+  <%= form_with url: 'update_settings', method: 'patch', model: @current_user, local: true do |f| %>
+    <p>
+      <%= f.label :display_name, t('students.display_name') %>
+      <%= f.text_field :display_name,
+                       value: @current_user.display_name.presence ||
```

Comment:
This `.presence || ...` shouldn't be necessary any more, instead just use `@current_user.display_name`